app/upgrades/v19/upgrades_test.go

```diff
@@ -163,3 +165,60 @@ func TestMigrateEthAccountsToBaseAccounts(t *testing.T) {
 		"expected different code",
 	)
 }
+
+func TestEnableCustomEIPs(t *testing.T) {
+	upgradeEIPs := []int64{0o000, 0o001, 0o002}
+
+	testCases := []struct {
+		name       string
+		activeEIPs []int64
+		expError   bool
+		expEIPsNum int
+	}{
+		{
+			name:       "repeated EIP - skip",
+			activeEIPs: []int64{0o000},
+			expError:   false,
+			expEIPsNum: 3,
+		},
+		{
+			name:       "all new EIP",
+			activeEIPs: []int64{3855},
+			expError:   false,
+			expEIPsNum: 4,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			network := testnetwork.NewUnitTestNetwork()
+
+			require.NoError(t, network.NextBlock(), "failed to advance block")
+
+			oldParams := network.App.EvmKeeper.GetParams(network.GetContext())
+			oldParams.ExtraEIPs = tc.activeEIPs
+			require.NoError(t, network.UpdateEvmParams(oldParams), "failed to update EVM params")
+			fmt.Println("EIPs: ", oldParams.ExtraEIPs)
+
+			logger := network.GetContext().Logger()
+			err := v19.EnableCustomEIPs(network.GetContext(), logger, network.App.EvmKeeper)
+			if tc.expError {
+				require.Error(t, err)
+			} else {
+				require.NoError(t, err)
+			}
```

Comment:
there are no error cases tested, so this could be simplified

```suggestion
			err := v19.EnableCustomEIPs(network.GetContext(), logger, network.App.EvmKeeper)
			require.NoError(t, err)
```